habitat-lab/habitat/config/benchmark/single_agent/pddl/zxz_move_test.yaml

```diff
@@ -0,0 +1,110 @@
+# The rearrangement task with robot_1 perceives the target object first.
+# Make sure the robot_0 is spot and the robot_1 is drone.
+objects:
+  - name: any_targets|0
+    expr_type: movable_entity_type
+  - name: TARGET_any_targets|0
+    expr_type: goal_entity_type
+  - name: robot_0
+    expr_type: robot_entity_type
+
+goal:
+  expr_type: AND
+  sub_exprs:
+    - at(any_targets|0,TARGET_any_targets|0)
+    - not_holding(robot_0)
+
+stage_goals:
+  stage_1:
+    expr_type: AND
+    sub_exprs:
+      - robot_at(any_targets|0, robot_0)
+      - holding(any_targets|0, robot_0)
+
+solution:
+  - move_backward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - turn_left(robot_0)
+  - turn_right(robot_0)
```

Comment:
What is this crazy file for? Can you explain a little bit? :astonished:

Reply:
This is aiming for locally test, sorry for uploading it. :(